browser_extension/css/style.css

```diff
@@ -1,74 +1,266 @@
-body {
-  background: #3d3d3d;
-  color: #fff;
+* {
+  box-sizing: border-box;
 }
 
-/* TOP BAR */
-.top-banner {
-  background: #454679;
-  color: #ffc900;
-  text-align: center;
-  padding: 5px;
+/* pacifico-regular - latin */
+@font-face {
+  font-family: 'Pacifico';
+  font-style: normal;
+  font-weight: 400;
+  src: local('Pacifico Regular'), local('Pacifico-Regular'),
+       url('../fonts/pacifico-v12-latin-regular.woff2') format('woff2'), /* Chrome 26+, Opera 23+, Firefox 39+ */
+       url('../fonts/pacifico-v12-latin-regular.woff') format('woff'); /* Chrome 6+, Firefox 3.6+, IE 9+, Safari 5.1+ */
 }
 
-/* LEFT SIDEBAR */
-.sidebar-heading {
-  background: #ffc900;
-  padding: 4px;
+html {
+  font-weight: 300;
+  -webkit-font-smoothing: antialiased;
+}
+/* width */
+::-webkit-scrollbar {
+  width: 10px;
 }
 
-.left-sidebar {
-  border: 1px solid #3d3d3d;
-  padding: 0;
+#onlineUserList {
+  color:white;
+}
+/* Track */
+::-webkit-scrollbar-track {
+  background: #E0E0E0; 
 }
 
-.left-sidebar li {
-  list-style: none;
+/* Handle */
+::-webkit-scrollbar-thumb {
+  background: #888; 
 }
 
-.left-sidebar h3 {
-  text-align: left;
+/* Handle on hover */
+::-webkit-scrollbar-thumb:hover {
+  background: #ffc900; 
 }
 
-#onlineUserColumn {
-  border: 2px solid black;
+.containerChat {
+  height: 100%;
+
+}
+.containerChat2 {
+  height: 100%;
+  box-shadow: 2px 2px 5px rgba(0,0,0,0.5),
+            inset 5px 5px 8px rgba(0,0,0,0.5);
 }
 
-#channelListHeader {}
+#logout {
+  background:red;
+}
+.headerTitle {
+  font-family: 'Pacifico'; 
+  color: #ffc900; 
+  text-align: center;  
+  font-size: 130%; 
+  padding-top: 3%;
+}
+.headerBack {
+  background: #7E57C2;
+  height: 6%;
+}
 
-.users_online {}
+html, input {
+  font-family:
+    "HelveticaNeue-Light",
+    "Helvetica Neue Light",
+    "Helvetica Neue",
+    Helvetica,
+    Arial,
+    "Lucida Grande",
+    sans-serif;
+}
 
-/* CHAT AREA */
-.chat-area {
-  padding-bottom: 22px;
-  background-color: #45467949;
-  max-height: 540px;
-  max-width: 100%;
-  overflow-y: scroll;
-  display: flex;
-  flex-direction: column-reverse;
+html, body {
+  height: 100%;
+  margin: 0;
+  padding: 0;
 }
 
-#chatContentArea li {
+ul {
   list-style: none;
+  word-wrap: break-word;
+}
+
+
+.chatPage {
+  height: 100%;
+  background-color:#E0E0E0;
+  box-shadow: 2px 2px 5px rgba(0,0,0,0.5),
+            inset 5px 5px 8px rgba(0,0,0,0.5);
 }
+/* Pages */
```

Comment:
Remove lines #96 - #127
The properties don't exist since login functionality has been moved to the popup